### ISSUE TITLE
error when setting up entry on 0.107.0b4

### ISSUE THREAD

**Issue by Diondk** [wxt9861/esxi_stats#23]
**Describe the bug**
I am testing my configuration on 0.107.0b4 and i am getting an error after a restart of HA.
Component doesnt initilize at all. 

```
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/config_entries.py", line 216, in async_setup
    hass, self
  File "/config/custom_components/esxi_stats/__init__.py", line 102, in async_setup_entry
    config[DOMAIN]["monitored_conditions"] = []
TypeError: 'mappingproxy' object does not support item assignment
```

**To Reproduce**
1. Join the beta channal
2. Upgrade to latest beta
3. restart HA
4. See error in the logs

**Expected behavior**
should be working in 0.107, i know this is a beta its more a headsup as in my production HA its working fine. 

**Screenshots**

**ESXi environment (please complete the following information):**
 - host version 6.7 full enterprise license

**Comment by wxt9861** [comment]
Thank you for reporting this. I've got it working, but let me do a bit more testing before releasing a fix. Probably tomorrow.

**Comment by Diondk** [comment]
I also got it working after commenting one line, it was just a temp fix. 

i commented line 102 in __init__.py
`    # config[DOMAIN]["monitored_conditions"] = []`

after that it started working, as i was only using this on my test machine i didnt investigate further.

**Comment by wxt9861** [comment]
Yep. That only solves 1 part of the problem. If you have the component already setup, commenting out that line fixes the issue.

However, if you try to add the component from the integrations page it will not add properly and will not pull any stats.

**Comment by wxt9861** [comment]
Beta release (0.5.2b3) with the fix is available. https://github.com/wxt9861/esxi_stats/releases/tag/0.5.2b3

**Comment by pergolafabio** [comment]
ah , thnx
but beta is not possible to install with HACS

**Comment by wxt9861** [comment]
You can install beta via HACS by going to the the integration in HACS, click the 3 dots to pull up more options menu and select Show Beta.

<img width="1327" alt="image" src="https://user-images.githubusercontent.com/34514130/76997922-150fa800-694c-11ea-9a85-c8a98a7f7bda.png">

**Comment by wxt9861** [comment]
Fixed in #25
